vllm/distributed/parallel_state.py

```diff
@@ -720,10 +721,12 @@ def init_world_group(ranks: List[int], local_rank: int,
 
 
 def init_model_parallel_group(
-        group_ranks: List[List[int]],
-        local_rank: int,
-        backend: str,
-        use_custom_allreduce: Optional[bool] = None) -> GroupCoordinator:
+    group_ranks: List[List[int]],
+    local_rank: int,
+    backend: str,
+    use_custom_allreduce: Optional[bool] = None,
+    use_mq_broadcaster: bool = False,
```

Comment:
Nit:
```suggestion
    use_message_queue_broadcaster: bool = False,
```
Also, how can a user turn this flag on?

Reply:
it is only used for tensor parallel now, where the broadcast happens.

Reply:
fixed in [daa49cc](https://github.com/vllm-project/vllm/pull/6183/commits/daa49cca65ce5ba2d44ec2b17de14582a12f7dd4)